generator/template_repo.go

```diff
@@ -580,3 +582,10 @@ func dict(values ...interface{}) (map[string]interface{}, error) {
 	}
 	return dict, nil
 }
+
+func quoteTagValue(value string) string {
+	if strings.Contains(value, "`") {
+		panic(fmt.Sprintf("Tag value contains backtick and conflicts with Go struct tags: %#v", value))
```

Comment:
is this #2423  ?

Reply:
I will remove this in favor of PrintTags